funnel/views/project.py

```diff
@@ -586,15 +595,31 @@ def cfp_transition(self) -> ReturnView:
             'error_description': _("Invalid form submission"),
         }
 
+    @route('register_modal', methods=['GET'])
+    @render_with('register_modal.html.jinja2')
+    @requires_login
+    def register_modal(self) -> ReturnRenderWith:
+        """Edit project banner."""
+        form = ProjectRSVPForm()
+        return {
+            'project': self.obj.current_access(datasets=('primary', 'related')),
```

Comment:
Do we need this full dataset for `project` for just the modal form? This can be reduced to a smaller dataset with just relevant fields.